packages/core/scss/color-system/_palettes.scss

```diff
@@ -0,0 +1,915 @@
+        200: #bcaaa4,
+        300: #a1887f,
+        400: #8d6e63,
+        500: #795548,
+        600: #6d4c41,
+        700: #5d4037,
+        800: #4e342e,
+        900: #3e2723,
+        1000: #000000
+    ),
+
+
+    // Primer
+    primerGray: (
+        0: #ffffff,
+        50: #f6f8fa,
+        100: #eaeef2,
+        200: #d0d7de,
+        300: #afb8c1,
+        400: #8c959f,
+        500: #6e7781,
+        600: #57606a,
+        700: #424a53,
+        800: #32383f,
+        900: #24292f,
+        1000: #000000
+    ),
+    primerBlue: (
+        0: #ffffff,
+        50: #ddf4ff,
+        100: #b6e3ff,
+        200: #80ccff,
+        300: #54aeff,
+        400: #218bff,
+        500: #0969da,
+        600: #0550ae,
+        700: #033d8b,
+        800: #0a3069,
+        900: #002155,
+        1000: #000000
+    ),
+    primerGreen: (
+        0: #ffffff,
+        50: #dafbe1,
+        100: #aceebb,
+        200: #6fdd8b,
+        300: #4ac26b,
+        400: #2da44e,
+        500: #1a7f37,
+        600: #116329,
+        700: #044f1e,
+        800: #003d16,
+        900: #002d11,
+        1000: #000000
+    ),
+    primerYellow: (
+        0: #ffffff,
+        50: #fff8c5,
+        100: #fae17d,
+        200: #eac54f,
+        300: #d4a72c,
+        400: #bf8700,
+        500: #9a6700,
+        600: #7d4e00,
+        700: #633c01,
+        800: #4d2d00,
+        900: #3b2300,
+        1000: #000000
+    ),
+    primerOrange: (
+        0: #ffffff,
+        50: #fff1e5,
+        100: #ffd8b5,
+        200: #ffb77c,
+        300: #fb8f44,
+        400: #e16f24,
+        500: #bc4c00,
+        600: #953800,
+        700: #762c00,
+        800: #5c2200,
+        900: #471700,
+        1000: #000000
+    ),
+    primerRed: (
+        0: #ffffff,
+        50: #ffebe9,
+        100: #ffcecb,
+        200: #ffaba8,
+        300: #ff8182,
+        400: #fa4549,
+        500: #cf222e,
+        600: #a40e26,
+        700: #82071e,
+        800: #660018,
+        900: #4c0014,
+        1000: #000000
+    ),
+    primerPurple: (
+        0: #ffffff,
+        50: #fbefff,
+        100: #ecd8ff,
+        200: #d8b9ff,
+        300: #c297ff,
+        400: #a475f9,
+        500: #8250df,
+        600: #6639ba,
+        700: #512a97,
+        800: #3e1f79,
+        900: #2e1461,
+        1000: #000000
+    ),
+    primerPink: (
+        0: #ffffff,
+        50: #ffeff7,
+        100: #ffd3eb,
+        200: #ffadda,
+        300: #ff80c8,
+        400: #e85aad,
+        500: #bf3989,
+        600: #99286e,
+        700: #772057,
+        800: #611347,
+        900: #4d0336,
+        1000: #000000
+    ),
+    primerCoral: (
+        0: #ffffff,
+        50: #fff0eb,
+        100: #ffd6cc,
+        200: #ffb4a1,
+        300: #fd8c73,
+        400: #ec6547,
+        500: #c4432b,
+        600: #9e2f1c,
+        700: #801f0f,
+        800: #691105,
+        900: #510901,
+        1000: #000000
+    ),
+
+
+    // Tailwind
+    tailwindNeutral: (
```

Comment:
do we really need those palettes ? I think we can refrain from adding tailwind and primer palettes here